.github/workflows/ibis-backends.yml

```diff
@@ -328,6 +346,25 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: delete header rows on ubuntu
+        if: matrix.os == 'ubuntu-latest' && matrix.backend.name == 'flink'
+        run: |
+          python -c "exec(\"TEST_FILES = ['functional_alltypes', 'diamonds', 'batting', 'awards_players']\nfor file in TEST_FILES:\n    with open(f'ci/ibis-testing-data/csv/{file}.csv', 'r+') as f:\n        f.readline()\n        data=f.read()\n        f.seek(0)\n        f.write(data)\n        f.truncate()\n\")"
```

Comment:
Actually, why not use the equivalent parquet files? It looks like flink supports ingesting those. That way we don't need to maintain any hacks to remove headers from CSV files :)

Reply:
> Can this be done in the `TestConf` class? As is, this will be challenging to maintain.

Yeah, I think this is possible, especially given I ended up using Python to do this in order to support Windows in the end.

> Actually, why not use the equivalent parquet files? It looks like flink supports ingesting those. That way we don't need to maintain any hacks to remove headers from CSV files :)

I believe Parquet requires some additional JAR to be installed; see https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/connectors/datastream/formats/parquet/. I'm sure it's very doable, but I tried a while back when doing the manual Flink tests, and never managed to get it set up properly.

So, my preference would be to move this into `TestConf`, if that's OK.

Reply:
I actually found I could just create tables from pandas dataframes, and I loaded those from parquet (as in the pandas backend `TestConf`).

Reply:
(I also couldn't think of a great way to do what I had originally in `TestConf`, since I didn't want to corrupt the local files, and wasn't sure if `TestConf` had a reliable post-run hook somewhere.)